miri

```diff
@@ -134,7 +134,8 @@ test|test-debug|bless|bless-debug)
     esac
     # Then test, and let caller control flags.
     # Only in root project as `cargo-miri` has no tests.
-    exec cargo test $CARGO_BUILD_FLAGS "$@"
+    cargo test $CARGO_BUILD_FLAGS "$@"
+    cargo test --manifest-path ui_test/Cargo.toml
```

Comment:
`$CARGO_BUILD_FLAGS "$@"` should probably be added here as well?

Reply:
I went all the way and implemented command line path filtering again

Reply:
Awesome, thanks. :)